### ISSUE TITLE
Radio Group direction should default to horizontal 

### ISSUE THREAD

**Issue by Alek99** [reflex-dev/reflex#4070]
Thinking about radio group most use cases are horizontal not vertical so direction should default as that

**Comment by Nazneenp** [comment]
i would love to  contribute. plz assign me

**Comment by Manojvbhat** [comment]
Hello @Alek99 ,
If I understand this correct, you want to change the radio group default direction to row in create method under HighLevelRadioGroup:
 `direction = props.pop("direction", "column")` 
to 
`direction = props.pop("direction", "row")`

**Comment by picklelo** [comment]
@Manojvbhat yes I think that's right. The only thing is this would have breaking changes to existing reflex apps... Not sure if we can just bring the change in right away, we may need to wait until 0.7.0 for a breaking change. But if you put up a PR we can review/have it ready for then!

**Comment by Manojvbhat** [comment]
Hi @picklelo ,
I raised a PR for this issue. It seems to have passed all the tests:
![image](https://github.com/user-attachments/assets/c7933241-0261-423d-accc-be298d26f2f5)
Documentation in the website needs to be changed as well, how can I do that?
